plugins/CoreHome/javascripts/dataTable.js

```diff
@@ -974,11 +987,16 @@ $.extend(DataTable.prototype, UIControl.prototype, {
             }
         });
 
+        $searchInput.on("blur", function () {
+            self.param.filter_trigger_id = 0;
```

Comment:
Should we maybe simply unset the parameter instead of setting a `0`?
```suggestion
            delete self.param.filter_trigger_id;
```

Reply:
Can do, it should not make any difference since we check whether it's not undefined before checking the value, so this should work as well.